packages/database/package.json

```diff
@@ -43,6 +43,7 @@
     "db-migrate-pg": "^1.2.2",
     "dotenv": "^8.2.0",
     "knex": "0.14.6",
+    "lodash": "^4.17.4",
```

Comment:
I think we typically just add a dependency for the required function from `lodash`, eg. `"lodash.cloneDeep": ...`

Reply:
Good point, updated